src/main/java/gregtech/loaders/recipe/chemistry/PolymerRecipes.java

```diff
@@ -470,4 +469,37 @@ private static void polybenzimidazoleProcess() {
                 .output(dust, ChromiumTrioxide, 4)
                 .buildAndRegister();
     }
+
+    public static void polycaprolactamProcess() {
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[HV]).duration(400)
+                .fluidInputs(Benzene.getFluid(1000), Hydrogen.getFluid(6000))
+                .fluidOutputs(Cyclohexane.getFluid(1000))
+                .notConsumable(new IntCircuitIngredient(1))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[LV]).duration(100)
```

Comment:
This chemical reaction can have all of its inputs and outputs divided by 2, such that you will have: `1000 Chlorine + 1000 Nitric Oxide -> 1000 Nitrosyl Chloride`.